lib/cryptoexchange/exchanges/bigone/market.rb

```diff
@@ -0,0 +1,8 @@
+module Cryptoexchange::Exchanges
+  module Bigone
+    class Market
+      NAME = 'bigone'
+      API_URL = 'https://api.big.one/markets'
```

Comment:
Best to have API_URL be `https://api.big.one` as markets may not been appended in all endpoints especially if we implement User Auth in the future